src/store/Transaction.ts

```diff
@@ -285,6 +293,12 @@ export default {
             const subscriptions: Observable<Transaction[]>[] = [];
             commit('isFetchingTransactions', true);
 
+            const multisigAccountGraphInfo: MultisigAccountInfo[][] = rootGetters['account/multisigAccountGraphInfo'];
```

Comment:
I saw the interface `account/multisigAccountGraphInfo` is created `MultisigAccountInfo[];` single array.

why over here is a 2D array?

Reply:
because it refers to Multisig entries which is a 2D array of multisigAccountInfo